packages/babel-types/src/builders/flow/createTypeAnnotationBasedOnTypeof.ts

```diff
@@ -40,6 +42,10 @@ export default function createTypeAnnotationBasedOnTypeof(
     return genericTypeAnnotation(identifier("Object"));
   } else if (type === "symbol") {
     return genericTypeAnnotation(identifier("Symbol"));
+  } else if (type === "bigint") {
+    // todo: use BigInt annotation when Flow supports BigInt
+    // https://github.com/facebook/flow/issues/6639
+    return anyTypeAnnotation();
```

Comment:
We can not use `numberTypeAnnotation` here because `Math` functions do not work with BigInt. Mark it as `any` as it is not supported in Flow.